features/filesystem/littlefs/littlefs/lfs.c

```diff
@@ -783,26 +783,19 @@ static int lfs_dir_find(lfs_t *lfs, lfs_dir_t *dir,
         lfs_entry_t *entry, const char **path) {
     const char *pathname = *path;
     size_t pathlen;
+    entry->d.type = LFS_TYPE_DIR;
+    entry->d.elen = sizeof(entry->d) - 4;
```

Comment:
Will ` sizeof(entry->d)` be ever < 4? Will that be corner case considering `elen` is uint8_t?

Reply:
No, the prefix for the entry stored on disk is a 4 byte header. If entry->d is less than 4, than the struct is malformed.

Also GCC is actually smart enough to warn on underflow for constants like these.